### ISSUE TITLE
Manaweave in a Bottle render issue in toolbar

### ISSUE THREAD

**Issue by GeorgeRandall** [Mithion/Mana-And-Artifice#196]
Four variants of Manaweave in a Bottle render incorrectly in the player's toolbar. Auctoro and Adnecto each render simultaneously on top of each other, and Relligo and Capistro also each render simultaneously  on top of each other. This only happens in the toolbar, and only when the player does not have an empty main-hand. Offhand items do not affect it. It renders correctly in inventory and as an item in the world.

I have other mods installed, but none I would expect to affect rendering.

**Comment by Mithion** [comment]
![image](https://user-images.githubusercontent.com/2482373/111909265-04987e80-8a33-11eb-9393-f2e85c1bd618.png)
Not seeing this in my dev environment.  Will need some more steps to reproduce.

**Comment by GeorgeRandall** [comment]
![screenshotBottleBug](https://user-images.githubusercontent.com/32231063/111919540-f1040c80-8a60-11eb-9692-723eabcfd296.png)

There's a screenshot of what I'm seeing. Poked around some more. Found a way to cause it to also apply in the inventory: If I am in creative mode, open various tabs, and scroll up and down, the issue comes and goes. For example in the Miscellaneous tab (lava bucket), it is present at the top, but when scrolled to the bottom it returns to normal. 

I started a new creative world and did nothing but give myself a mana wand and bottles, and bottle up some mana weaves and it still happened. 
![screenshotBottleBug2](https://user-images.githubusercontent.com/32231063/111920345-c3b95d80-8a64-11eb-9785-540f0b2cbce9.png)

I'll start start eliminating mod interactions when I get the chance.
